### PR TITLE
Update es.m3u

### PR DESCRIPTION
**Pull request by trancer1994**

### CODE REVIEW

**Comment by themak1985** [comment]
@trancer1994 https://cdn01.yowi.tv/4131RI73I9/master.m3u8 does not work and there are some conflicts with master, but they look easily resolve-able.

File: channels/es.m3u [hunk 1]
```diff
@@ -663,3 +663,15 @@ https://livelist01.yowi.tv/lista/81c601f370e44dc566113fd752204be5f5f53b61/master
...
+#EXTINF:-1 tvg-id="" tvg-name="Lleida Televisio" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/6qZNM6E.png" group-title="",Lleida Televisio
+https://cdn01.yowi.tv/lleida/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="http://directostv.teleame.com/wp-content/uploads/2016/02/Canal-45-Ja%C3%A9n.png" group-title="",CANAL 45 TV
+https://cdn01.yowi.tv/503L6OKTE2/live-1000.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Canal 4 Mancha Centro
+https://5924d3ad0efcf.streamlock.net/canal4/canal4live/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/MCHSRB1.png" group-title="",Canal 4
+http://149.202.68.178:1935/live/peprecasens/manifest.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/87KGlQz.png" group-title="",Campiña Sur TV
+https://cdn01.yowi.tv/4131RI73I9/master.m3u8
```
Comment (LaneSh4d0w) [inline channels/es.m3u:675]: This channel does not seem to appear on Yowi TV. Where did you get it from ?

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]

**Review by freearhey — CHANGES_REQUESTED** [review]
https://github.com/iptv-org/iptv/pull/2923#issuecomment-850062642

**Comment by LaneSh4d0w** [comment]
Ah, this channel works now... Not 24/7 ?

**Comment by themak1985** [comment]
> Ah, this channel works now... Not 24/7 ?

Seems like it - now working for me too.

**Comment by LaneSh4d0w** [comment]
I'll think i'll resolve the conflict and add the Non 24-7 tag.

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**